xml/schema/CRM/Funding/FundingApplicationCiviOfficeTemplate.xml

```diff
@@ -0,0 +1,82 @@
+<?xml version="1.0" encoding="iso-8859-1" ?>
+
+<table>
+  <base>CRM/Funding</base>
+  <class>ApplicationCiviOfficeTemplate</class>
+  <name>civicrm_funding_application_civioffice_template</name>
+  <title>Application Template</title>
+  <titlePlural>Application Templates</titlePlural>
+  <comment>Templates for use in application portal</comment>
+  <log>true</log>
+
+  <field>
+    <name>id</name>
+    <type>int unsigned</type>
+    <required>true</required>
+    <comment>Unique FundingApplicationCiviOfficeTemplate ID</comment>
+    <html>
+      <type>Number</type>
+    </html>
+  </field>
+  <primaryKey>
+    <name>id</name>
+    <autoincrement>true</autoincrement>
+  </primaryKey>
+
+  <field>
+    <!--
+      Without "funding_" so the generated constraint name does not exceed the
+      limit of 64 chars.
+    -->
+    <name>case_type_id</name>
```

Comment:
Oh that's too bad, but `case` without `funding` might be ambiguous with *CiviCase* entities. Will at least an `f` fit in …?

Reply:
Actually I'm not sure if `f_case_type_id` (or `fcase_type_id`) would be less confusing... At least to me it looks strange. I'd think it's clear from the context that it has no relation to `CiviCase`.